diff/diff_test.go

```diff
@@ -496,6 +496,60 @@ func TestParseMultiFileDiffHeaders(t *testing.T) {
 				},
 			},
 		},
+		{
+			filename: "sample_contains_added_deleted_files.diff",
+			wantDiffs: []*FileDiff{
+				{
+					OrigName: "source_a/file_1.txt",
+					OrigTime: nil,
+					NewName:  "source_b/file_1.txt",
+					NewTime:  nil,
+					Extended: []string{
+						"diff -u source_a/file_1.txt  source_b/file_1.txt",
+					},
+				},
+				{
+					OrigName: "source_a/file_2.txt",
+					OrigTime: nil,
+					NewName:  "",
+					NewTime:  nil,
+					Extended: nil,
+				},
+				{
+					OrigName: "source_b/file_3.txt",
+					OrigTime: nil,
+					NewName:  "",
+					NewTime:  nil,
+					Extended: nil,
+				},
+			},
+		},
+		{
+			filename: "sample_contains_only_added_deleted_files.diff",
+			wantDiffs: []*FileDiff{
+				{
+					OrigName: "source_a/file_1.txt",
+					OrigTime: nil,
+					NewName:  "",
+					NewTime:  nil,
+					Extended: nil,
+				},
+				{
+					OrigName: "source_a/file_2.txt",
+					OrigTime: nil,
+					NewName:  "",
+					NewTime:  nil,
+					Extended: nil,
+				},
+				{
+					OrigName: "source_b/file_3.txt",
```

Comment:
maybe make one of the 3 files be added, not all of them deleted? Or split the test case into two:
* One where files are only added
* One where files are only deleted

Reply:
Please see PR description. All filenames from `Only in {source}: {filename}` messages will be added to `OrigName` field.